js/templates/modals/orderDetail/summaryTab/orderDetails.html

```diff
@@ -50,7 +50,7 @@ <h2 class="tx4 margRTn"><%= ob.polyT('orderDetail.summaryTab.orderDetails.headin
           <% if (ob.order.shipping && ob.order.shipping.country !== 'NA') { %>
             <div><%= ob.order.shipping.shipTo %></div>
             <% if (ob.order.shipping.address) { %>
-            <div><%= ob.order.shipping.address %></div>
+            <div><%= ob.order.shipping.address.replace(/, /g, ',<br>') %></div>
```

Comment:
Good point. Technically I think the comma is supposed to be there, but it looks weird.